src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,94 @@
+- Hack X - 1-3 of the Miscellaneous Interactions machines is picked to be hacked by the Space Ninja. Note that you can still hack machines that aren't included in your objectives. (Replaces "Set Everyone to Wanted")
+
+## Miscellaneous Interactions
+
+- Hacking the criminal records computer is now one of the many miscellaneous actions the Space Ninja can do in order to frustrate the crew - The error announcement will now only be sent by the criminal records computer on the Security channel, opposed to the previous station-wide Central Command Announcement.
+- The crew monitoring server is now prone to being hacked by the Space Ninja, doing so will disable everyone's sensors until deconstructed and reconstructed.
+- The station records computer is now prone to being hacked by the Space Ninja, doing so will delete a small % of random personnel from it accompanied by a station-wide announcement.
+- The camera router is now prone to being hacked by the Space Ninja, doing so will kill a large % of its connected cameras and render them useless.
+
+# Balance Changes
+
+Changes mostly geared towards fixing previously outright broken mechanics, as well to accomodate the new additions.
+
+## Nerfs
+
+These nerfs are intended to make previous go-to combat tactics less prevalent, hence incentivizing a more varied playstyle.
+
+- [Fix ninja stunlocking](https://github.com/space-wizards/space-station-14/pull/33244)
+- All Space Ninjas are now bio-engineered humans, making it so they don't get specie-specific attributes such as tail-dragging.
+- Removes the extended-capacity survival box from the ninja's satchel, as there's no longer any use for it.
+- Lower the energy katana's one-handed damage.
+- Lower the ninja throwing star's piercing damage to compensate for the newly added ninja caltrops.
+- Lower the ninja throwing star's stamina damage.
+
+## Buffs
+
+Enhances some things on the Space Ninja's kit in order to make it less of a one-trick pony, as well as to fix some inconsistencies.
+
+- Space Ninjas no longer have a need to eat nor drink.
+- Space Ninjas now come outfitted with a ninja headset as per [Alwayswannahunt's PR](https://github.com/space-wizards/space-station-14/pull/32841) , green encryption key included.
+- The space ninja gloves can now unlock access-restricted lockers and crates.
+- The energy katana's slash rate is slightly faster than it was previously when one-handed to compensate for it's damage loss.
+- The ninja throwing stars can now be stacked, as well as thrown in spreads.
+- When phasing, the Space Ninja is able to discreetly steal from people.
+
+## Miscellaneous Tweaks
+
+Mostly new additions that weren't either complete buffs nor complete nerfs.
+
+- The katana dash is considerably quieter, at the cost of a longer recharge duration.
```

Comment:
Im ok with this, as it gives the Ninja more of a chance to slip up and make a mistake. Right now a really good ninja is untouchable by security because of their amazing ability to pick fights. However, despite this change, I still think a very robust player can still escape 99% of situations.

Reply:
> Im ok with this, as it gives the Ninja more of a chance to slip up and make a mistake. Right now a really good ninja is untouchable by security because of their amazing ability to pick fights. However, despite this change, I still think a very robust player can still escape 99% of situations.

this is more as a way to give the ninja a proper quieter way to move around, this time along the dashes are made to be more silent, as the "loud" action is the EMAG (as it's also the more overpowered one), so the ninja can pick between either having AA access but going loud, or dashing from window to window stealthily